runtime/src/tests/work-item/work-item-list/page-objects/work-item-list.page.js

```diff
@@ -151,8 +151,8 @@ class WorkItemListPage {
     return parentElement.element(by.css('.workItemList_Delete'));
   }
   workItemKebabAssocateIterationButton (parentElement) {
-    browser.wait(until.presenceOf(parentElement.element(by.css('.workItemList_Iteration'))), constants.WAIT, 'Failed to find clickWorkItemKebabButton');
-    return parentElement.element(by.css('.workItemList_Iteration'));
+    browser.wait(until.presenceOf(parentElement.element(by.id('workItemList_Iteration'))), constants.WAIT, 'Failed to find clickWorkItemKebabButton');
```

Comment:
This line crosses the the maximum number of column standard, please break it in 2-3 lines.

Reply:
Why is this an ID ? This is an iterative component. I thought we decided on not to use ID on iterative component.